src/Bridges/Constraint/ltgt_to_interval.jl

```diff
@@ -0,0 +1,73 @@
+# The code here is mostly copied from the flip_sign.jl code for FlipSignBridge and GreaterToLessBridge
+
+"""
+    AbstractToIntervalBridge{T, S1, F}
+
+Bridge a `F`-in-`S1` constraint into an `F`-in-`Interval` constraint where we have either:
+* `S1 = MOI.GreaterThan{T}`
+* `S1 = MOI.LessThan{T}`
+
+The `F`-in-`S1` constraint is stored in the `constraint`
+field by convention.
+"""
+abstract type AbstractToIntervalBridge{
+    T, S1<:MOI.AbstractSet,
+    F<:MOI.AbstractFunction} <: SetMapBridge{T, MOI.Interval{T}, S1, F, F} end
+
+# The function map is the identity. It is also an involution, symmetric, and a symmetric involution.
+map_function(::Type{<:AbstractToIntervalBridge{T}}, func) where {T} = func
+inverse_map_function(BT::Type{<:AbstractToIntervalBridge}, func) = func
+adjoint_map_function(BT::Type{<:AbstractToIntervalBridge}, func) = func
+inverse_adjoint_map_function(BT::Type{<:AbstractToIntervalBridge}, func) = func
+
+# FIXME are these modify functions necessary?
+function MOI.modify(model::MOI.ModelLike, bridge::AbstractToIntervalBridge,
+                    change::MOI.ScalarCoefficientChange)
+    MOI.modify(
+        model, bridge.constraint,
+        MOI.ScalarCoefficientChange(change.variable, change.new_coefficient))
```

Comment:
You can pass `change` as it is, no need to build a new one

Reply:
So I remove the function altogether?